kevm-pyk/src/kevm_pyk/cli.py

```diff
@@ -61,20 +61,20 @@ def kprove_args(self) -> ArgumentParser:
             dest='always-check_subsumption',
             default=True,
             action='store_true',
-            help='Check subsumption even on non-terminal nodes.',
+            help='Check subsumption even on non-terminal nodes (default)(experimental).',
```

Comment:
I'd suggest to separate `default` and `experimental` with a comma instead:
```suggestion
            help='Check subsumption even on non-terminal nodes (default, experimental).',
```